docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+.. _kinto-glossary:
+
+Glossary
+########
+
+
+.. glossary::
+
+    Cliquet Protocol
+            A system of rules that explains the way to interact with Kinto,
+            and how its data is organized.
+
+    HTTP API
+            The definition of a set of HTTP request messages along with the
```

Comment:
This is the definition of.. Web Resource ;)  For the definition of HTTP API it's closer to https://en.wikipedia.org/wiki/Web_API.

So what about:

multiple publicly exposed endpoints that accept HTTP requests and respond with the requested data, in the form of JSON, to interact with the data held in the Kinto Server